lib/caesar.rb

```diff
@@ -1,6 +1,24 @@
+require "pry"
 class Caesar
-  attr_reader :key
+
 
   def initialize
+
+  end
+
+
+  def caesar_cipher(string, shift)
```

Comment:
Make sure you are following the spec. It should be eng_to_cipher(text, rotation)